### PR TITLE
chore: 162 update to react 18 and pf5

### PR DESCRIPTION
**Pull request by gitdallas**

### CODE REVIEW

**Comment by gitdallas** [comment]
Need to get rid of enzyme and react-router-dom

**Review by wise-king-sullyman — APPROVED** [review]

**Review by jenny-s51 — COMMENTED** [review]
Component updates look good but it looks like some styles and fonts are missing

Before:
Screenshot 2023-08-21 at 2 19 13 PM

After:
Screenshot 2023-08-21 at 2 21 48 PM

**Review by jenny-s51 — APPROVED** [review]
LGTM following latest changes!

**Review by nicolethoen — COMMENTED** [review]
What kinds of errors does npm install produce that force us to need to use `npm install --force`?

**Comment by gitdallas** [comment]
@nicolethoen  - it wasn't allowing me to install without it. i know there have been changes with npm install over the past couple versions that make it so it will exit out on things that it previously wouldn't have without the force.

i'm unsure if this has changed since i first worked on this pr.. i'll check.

File: test-setup.js
Comment (jenny-s51) [inline test-setup.js:1]: Can we remove this unused file @gitdallas

Comment (gitdallas) [reply test-setup.js:1]: I believe so, it looks like some remnants of enzyme.

**Review by jenny-s51 — APPROVED** [review]
Thank you @gitdallas - LGTM

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**